.github/workflows/tests_linux.yml

```diff
@@ -184,7 +184,8 @@ jobs:
         run: |
           cd main/
           DEVICENAME=`echo ${{ matrix.pl_backend }} | sed "s/_/./g"`
-          PL_DEVICE=${DEVICENAME} python -m pytest tests/ $COVERAGE_FLAGS
+          PL_DEVICE=${DEVICENAME} python -m pytest tests/ -k "not test_native_mcm" $COVERAGE_FLAGS
+          OMP_NUM_THREADS=1 PL_DEVICE=${DEVICENAME} python -m pytest -n auto tests/test_native_mcm.py $COVERAGE_FLAGS
```

Comment:
But you are making ` OMP_NUM_THREADS=1`, isn't the number of parallel cores in pytest controlled with the OMP_NUM_THREADS variable?

Reply:
`OMP_NUM_THREADS=1` sets the number of OpenMP threads per pytest worker to `1`, then `-n` sets the number of pytest workers. So `OMP_NUM_THREADS=1` is to avoid oversubscription.

Reply:
Hmm. Nice to know. Thank you for clarifying.